Makefile

```diff
@@ -21,7 +21,7 @@ rmcache:
 
 
 mypy: .develop
-	mypy multidict
+	mypy multidict tests/test_mypy.py
```

Comment:
It's in the pull request description:
The commit add the necessary type definitions to make all the classes
non-abstract and **adds a "test" to make sure these type definitions can
actually be used**.

(this is the type-checking test, which is outside the multidict module obviously)